### ISSUE TITLE
Warn if multiple parcel processes are running

### ISSUE THREAD

**Issue by poppinlp** [parcel-bundler/parcel#1858]
# 🐛 bug report

1. Run `npx parcel watch index.html`
1. Run `npx parcel build index.html`
1. Then modified something to trigger hot reload in `watch` mode
1. The modification seems not work

## 🎛 Configuration (.babelrc, package.json, cli command)

```js
// my .babelrc
{
  "presets": [
    ["env", {
      "modules": false,
      "targets": {
        "browsers": ["last 2 Chrome versions"]
      }
    }]
  ]
}
```

## 🤔 Expected Behavior

Build result could be right.

## 😯 Current Behavior

Seems it stays on the step 2 `build` result. But the hot reload trigger successfully.

## 🔦 Context

Not too much but sometimes makes me feel weird.

## 💻 Code Sample

A file named `index.html` which includes a `main.js`.
That js import some methods from `foobar.js`.
And the modification in step 4 is "remove or add some import methods in `main.js`".

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.9.7
| Node             | 10.8.0
| npm/Yarn         | npm 6.2.0
| Operating System | macOS 10.13.6

**Comment by DeMoorJasper** [comment]
Why would you run `build` after `watch`?

Build is for production, so running build after watch replaces the bundle with ones without the hot module reload code.

watch also builds the code, but adds certain parts for hot module reloading.

**Comment by poppinlp** [comment]
Yep. I see that `watch` builds code successfully. But the result has not changed which confused me.

Actually it's my mistake that i have opened `watch` mode in a tmux window and i forgot it.
Then i run `build` in another for some reason and then come back to `watch` window.
So... you know... i spend some time to find out why modification not work.

So i don't think it's a bug. But could there be some warnings when try to run `build` with a `watch` worker running?

**Comment by DeMoorJasper** [comment]
I guess it can be added, not sure if node has that much permissions though.
As this would be going into running processes and searching for parcel.

I'll change this into feature request
